pkg/harvester/components/UpgradeInfo.vue

```diff
@@ -26,15 +31,14 @@ export default {
     <Banner color="warning">
       <div>
         <strong>{{ t('harvester.upgradePage.upgradeInfo.warning') }}:</strong>
-        <p v-clean-html="t('harvester.upgradePage.upgradeInfo.doc', {}, true)" class="mb-5">
-        </p>
+        <p v-clean-html="t('harvester.upgradePage.upgradeInfo.doc', {url: upgradeLink}, true)" class="mb-5"></p>
```

Comment:
`target="_blank"` is ignore when giving translation key via `v-clean-html`. (not open link in new tab)
It's resolved in rancher but not harvester dashboard..

PR: https://github.com/rancher/dashboard/pull/9928